feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -253,6 +253,132 @@ openconfig-network-instance:
 * Generate traffic from ATE port 1
 * Validate ATE port 2 receives the innermost IPv4 traffic with correct VLAN and inner_decap_ipv6
 
+### TE-18.1.5 Rewrite the ingress innner packet TTL = 1, if the incoming TTL = 1.
+Create AFT entries using gRIBI to match on IP and TTL = 1 to
+encapsulate the matching packets in MPLS in UDP with inner ip ttl as 1.
+
+#### gRIBI RPC content
+
+The gRIBI client should send this proto message to the DUT to create AFT
+entries.  See [OC AFT Encap PR in progress](https://github.com/openconfig/public/pull/1153)
+for the new OC AFT model nodes needed for this.  
+
+```proto
```

Comment:
Please revise the format to use what is in TE-18.1.1 (proto specific format)

ref: https://github.com/openconfig/featureprofiles/pull/3656/files#diff-a6b35105c1accdb1889d9c529f4a13d6844d4a9844a90f3d3f736eba728502c5R48-R78